packages/wallet/tests/wallet.spec.ts

```diff
@@ -829,8 +833,8 @@ describe('Wallet integration', function () {
         const signed_3 = await w3_3.eth.signTransaction(transaction)
 
         const full_signed = {
-          raw: aggregate(signed_1.raw, signed_2.raw, signed_3.raw), // TODO: 'aggregate' name is too vague
-          tx: signed_1.tx
+          ...signed_1,
+          signature: aggregate(signed_1.signature, signed_2.signature, signed_3.signature) // TODO: 'aggregate' name is too vague
```

Comment:
how about `joinSignatures()` ? .. and btw, what kind of "signatures" are these? maybe we can make it more specific then, ie. `joinTransactionSignatures()`

Reply:
`aggregate` supports any kind of signature, it doesn't matter if it's a transaction signature or a message signature. I think renaming it to `joinSignatures` works